site/app/views/NavigationView.php

```diff
@@ -139,6 +139,29 @@ public function showGradeables($sections_to_list) {
         <tbody id="{$lower_title}_tbody">
 HTML;
             foreach ($gradeable_list as $gradeable => $g_data) {
+        
+                if ($g_data->getActiveVersion() == -1){
+                    $submission_status = array(
+                        "SUBMITTED" => "<em style='font-size: .8em;'>NOT SUMBITTED</em><br>",
+                        "AUTOGRADE" => ""
+                    );
+                }
+                else{
+                    if ($g_data->getTotalNonExtracreditAutograderPoints() == array()){
```

Comment:
What would cause this if statement to be entered? We have "floatval" around all casts on points when we have them, otherwise it'd be 0. 

The status of a submission is:
1. Not submitted (active version is -1)
2. Submitted, but cancelled (active version is 0)
3. Submitted and scored (active version is > 0)

Where the text for a button would be:
1. NOT SUBMITTED
2. CANCELLED
3. SUBMITTED (## / ##)

with an if/else if/else of:
```
if ($g_data->getActiveVersion() === -1) {
    // not submitted
}
else if ($g_data->getActiveVersion === 0) {
    // submitted but cancelled
}
else {
    // submitted and graded
}
```